src/daemon/compilation_daemon.cc

```diff
@@ -243,6 +246,8 @@ bool CompilationDaemon::SearchSimpleCache(
   if (!cache_) {
     return false;
   }
+  perf::Counter<perf::StatReporter, true> counter(
```

Comment:
`perf::Counter<perf::StatReporter>` should be fine everywhere, since the template argument is `true` by default.